test/unitTests/logging/TelemetryObserver.test.ts

```diff
@@ -27,20 +27,20 @@ suite('TelemetryReporterObserver', () => {
             sendTelemetryEvent: (eventName: string, properties?: { [key: string]: string }, measures?: { [key: string]: number }) => {
                 name += eventName;
                 property = properties;
-                measure.push(measures);
+                measure.push(measures!);
             },
             sendTelemetryErrorEvent: (eventName: string, properties?: { [key: string]: string; }, measures?: { [key: string]: number; }, errorProps?: string[]) => {
                 name += eventName;
                 property = properties;
-                measure.push(measures);
-                errorProp.push(errorProps);
+                measure.push(measures!);
+                errorProp.push(errorProps!);
             },
         };
     }, useModernNet);
 
     setup(() => {
         name = "";
-        property = null;
+        property = undefined;
```

Comment:
Could we have just set property to `{}` and avoided introducing `undefined`?

Reply:
In some cases it works, in others the code relies on undefined.  I'll check this one